contracts/lib/platform/EthAdapter.sol

```diff
@@ -66,10 +66,10 @@ contract EthAdapter is ERCDotFactory {
         //burn dot backed token
         tok.burnFrom(wallet, quantity);
         //send wallet eth
-        wallet.transfer(reserveCost * adapterRate);
+        address(wallet).transfer(reserveCost * adapterRate);
```

Comment:
compiler throwing issues to wallet.transfer

casted wallet to address(wallet) to avoid compiler error